contrib/cirrus/runner.sh

```diff
@@ -233,6 +233,17 @@ function _run_consistency() {
     SUGGESTION="run 'make generate-bindings' and commit all changes" ./hack/tree_status.sh
     make completions
     SUGGESTION="run 'make completions' and commit all changes" ./hack/tree_status.sh
+    if  [[ -z "$CIRRUS_TAG" ]] && \
+        req_env_vars CIRRUS_CHANGE_IN_REPO CIRRUS_PR DEST_BRANCH &> /dev/null
+    then
+        local base diffs
+        base=$(git merge-base $DEST_BRANCH $CIRRUS_CHANGE_IN_REPO)
+        diffs=$(git diff $base $CIRRUS_CHANGE_IN_REPO)
+        if egrep -q '^\+.+io/ioutils' || egrep -q '^\+.+ioutils\..+'; then
+            die "Found attempted use of deprecated ioutils:
+$diffs"
```

Comment:
echoing the full diffs is going to be rather daunting on a revendor or other big PR